subjects/macro_map/README.md

```diff
@@ -0,0 +1,50 @@
+## macro_map
+
+### Instructions
+
+Create a macro rule called `hash_map` to initialize and declare an `HashMap` at the same time, very similar to what `vec!` macro does for `Vector`.
+
+> Your macro should accept both leading commas and non leading commas syntax to be more flexible in terms of coding style and reflect the language general style.
```

Comment:
can be rephrase to `...both leading and non leading commas...` to avoid repetition.

Reply:
done